internal/promotion/runner/builtin/git_pr_waiter.go

```diff
@@ -13,6 +13,12 @@ import (
 	"github.com/akuity/kargo/pkg/promotion"
 	promoPkg "github.com/akuity/kargo/pkg/promotion"
 	"github.com/akuity/kargo/pkg/x/promotion/runner/builtin"
+
+	_ "github.com/akuity/kargo/internal/gitprovider/azure"  // Azure provider registration
+	_ "github.com/akuity/kargo/internal/gitprovider/bitbucket" // Bitbucket provider registration
+	_ "github.com/akuity/kargo/internal/gitprovider/gitea"  // Gitea provider registration
+	_ "github.com/akuity/kargo/internal/gitprovider/github" // GitHub provider registration
+	_ "github.com/akuity/kargo/internal/gitprovider/gitlab" // GitLab provider registration
```

Comment:
I don't think we need these here. `git_pr_opener.go` has it covered and they're both in the same package.

```suggestion
```

Reply:
I added this because I do not like the assumption of `git_pr_opener.go` existing, this makes things explicit.